src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+public class Deadline extends Task {
+    protected String deadline;
+
+    public Deadline(String description, int index, String deadline) {
+        super(description, index);
+        this.type = "D";
```

Comment:
can consider using constants (or Enum) for the type

Reply:
Noted